compio-driver/src/lib.rs

```diff
@@ -465,6 +469,21 @@ impl ProactorBuilder {
         self.pool_builder.create_or_reuse()
     }
 
+    /// Set `io-uring` sqpoll idle milliseconds, when `sqpoll_idle` is set,
+    /// io-uring sqpoll feature will be enabled
+    ///
+    /// # Notes
+    ///
+    /// Only effective when the `io-uring` feature is enabled
+    pub fn sqpoll_idle(&mut self, idle: u32) -> &mut Self {
```

Comment:
If it's time related, use `Duration`

Reply:
Duration is not inappropriate, sqpoll idle time unit is ms, if idle is a `Duration`, user can create a 1ns `Duration`

Reply:
unless check the `Duration` in run-time, if idle < 1ms, then panic

Reply:
Fare enough